packages/fluentui/react-northstar/src/components/Pill/Pill.tsx

```diff
@@ -133,6 +147,14 @@ export const Pill: ComponentWithAs<'span', PillProps> & FluentComponentStaticPro
         ...unhandledProps,
       })}
     >
+      {!!image &&
+        createShorthand(PillImage, image, {
+          defaultProps: () => ({ size }),
+        })}
```

Comment:
```suggestion
      {createShorthand(PillImage, image, {
          defaultProps: () => ({ size }),
        })}
```

It's redundant as `createShorthand()` handles it